embedding/cpp/include/elm_flutter_view.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_VIEW_H_
+#define FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_VIEW_H_
+
+#include <Elementary.h>
+#include <flutter/plugin_registry.h>
+#include <flutter_tizen.h>
+
+#include <string>
+#include <vector>
+
+// The app base class which creates and manages the Flutter engine instance.
+class ElmFlutterView : public flutter::PluginRegistry {
+ public:
+  virtual ~ElmFlutterView() {}
+
+  FlutterDesktopPluginRegistrarRef GetRegistrarForPlugin(
+      const std::string &plugin_name) override;
+
+  bool RunFlutterEngine(Evas_Object *parent);
+
+  bool RunFlutterEngine(Evas_Object *parent, int32_t width, int32_t height);
+
+  bool IsRunning() { return engine_ != nullptr; }
+
+  Evas_Object *evas_object() { return evas_object_; };
+
+  void Resize(int32_t width, int32_t height);
+
+  int32_t width() { return width_; };
+
+  int32_t height() { return height_; };
```

Comment:
I deleted them. Because as you say, it may return incorrect value.
If no initial size is specified when creating `ElmFlutterView`, 0 is returned.
I will think more about these accessors and add an appropriate API to engine code together.

Reply:
Great. Thanks!